### DISCUSSION TITLE
French Gui

### DISCUSSION THREAD

**Discussion by FranciumSoftware** [birdnet-team/BirdNET-Analyzer#425] [General]
Hello,
I have traslated the GUI of BirdNet in french. Tou can view that here.
[Fr.json](https://github.com/user-attachments/files/16762523/Fr.json)

**Comment by max-mauermann** [comment]
Hi,
thanks for contributing this translation!
I will create a pull request to include this in the repo

**Comment by FranciumSoftware** [comment]
I have tested that on the app and there is a bug: after analyze, the result table return in English.
![BirdNet Gui](https://github.com/user-attachments/assets/a8276050-35bb-48b2-a1ab-337793e02313)

**Comment by max-mauermann** [comment]
you are right, although this is a general problem with all translations.
I think this occurs, because a csv output file is created (which always has the english headers), which is then returned to the gradio component.
I will look into this.
